src/website/app/demos/Avatar/bestPractices.js

```diff
@@ -39,8 +39,8 @@ export default [
   },
   {
     type: 'dont',
-    description: `To prevent duplicate information, hide Avatar from Assistive
-Technology (AT) with \`ariaHidden="true"\` when the associated text is
+    description: `To prevent duplicate information, hide Avatar from assistive
+technologies with \`ariaHidden="true"\` when the associated text is
```

Comment:
@kylegach do you think we should link to the W3 definition of assistive technologies everywhere we mention it? Probably not necessary, but might help inform our audience: https://www.w3.org/WAI/PF/aria/terms#def_at

Reply:
Not a bad idea.

Reply:
I believe it should be `aria-hidden` rather than `ariaHidden`.

Reply:
Aren't both true? camelCase when in JSX and hypenated when in a JS object? Not sure how we disambiguate that. Maybe sticking with what HTML uses is best, though.

Reply:
No, `ariaHidden` does not work in JSX.